### ISSUE TITLE
When performing a fresh install of Craft with an existing `config/project` folder, Hyper causes the "applying existing project config" step to fail

### ISSUE THREAD

**Issue by kylecotter** [verbb/hyper#46]
### Describe the bug

When installing Craft from scratch, but including an existing `config/project` folder as part of the install, (E.g. custom bootstrap/boilerplate starting point), when the Craft installer gets to the `apply existing project config` step, it fails, because Hyper is trying to do something in its `FieldCache` service with regards to `$oldHandle`

```
Exception: verbb\hyper\services\FieldCache::updateContentField():
Argument #2 ($oldHandle) must be of type string, null given,
called in /var/www/vendor/verbb/hyper/src/services/FieldCache.php on line 50 (/var/www/vendor/verbb/hyper/src/services/FieldCache.php:87)
```

If I had to guess, I'd think this should also apply in this Craft install scenario too https://github.com/verbb/hyper/blob/4c183bb686669302106153949bc51dea96ea37c2/src/services/FieldCache.php#L25-L27

### Steps to reproduce

1. Have an existing `config/project` containing a Hyper field in place within a new Craft CMS project folder structure
2. Run `./craft install` and follow the prompts
3. Craft should attempt to apply the existing project config
4. It fails because Hyper is attempting to run it's caching service when it shouldn't

### Craft CMS version

4.4.5

### Plugin version

1.1.2

### Multi-site?

No

**Comment by engram-design** [comment]
Should be fixed for the next release. To get this early run `composer require verbb/hyper:"dev-craft-4 as 1.1.2"`

**Comment by engram-design** [comment]
Fixed in [1.1.3](https://github.com/verbb/hyper/releases/tag/1.1.3)
